README.md

```diff
@@ -25,15 +25,25 @@ Our Android template: **[template](https://github.com/nimblehq/android-templates
 
 4. Update `android_version_code` and `android_version_name` in `template/build.gradle`
 
+## Wiki
+
+Learn more about our Android templates on the [Wiki](https://github.com/nimblehq/android-templates/wiki).
+
+## License
+
+This project is Copyright (c) 2014 and onwards. It is free software,
+and may be redistributed under the terms specified in the [LICENSE] file.
+
+[LICENSE]: /LICENSE
+
 ## About
 
-<a href="https://nimblehq.co/">
-  <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://assets.nimblehq.co/logo/dark/logo-dark-text-160.png">
-    <img alt="Nimble logo" src="https://assets.nimblehq.co/logo/light/logo-light-text-160.png">
-  </picture>    
-</a>
+![Nimble](https://assets.nimblehq.co/logo/dark/logo-dark-text-160.png)
 
 This project is maintained and funded by Nimble.
 
-Learn more about our Android templates on the [Wiki](https://github.com/nimblehq/android-templates/wiki).
+We love open source and do our part in sharing our work with the community!
+See [our other projects][community] or [hire our team][hire] to help build your product.
+
+[community]: https://github.com/nimblehq
+[hire]: https://nimblehq.co/
```

Comment:
Lack of the last empty line 🙏🏼

Reply:
which empty line @chornerman? 🤔 It think it's fine 💭

Reply:
@chornerman I copy from iOS template. 🤔

Reply:
@luongvo @lydiasama You can check out the branch and see that there's no last blank line which normally we always include in any type of file. However, GitHub seems to not show any warning. Does it mean it's okay for this case? 🤔 If so, I think we can ignore this comment. No biggie here.

Reply:
@chornerman You mean the end of line or before the [hire]?

Reply:
Yes, sorry for the confusing word 😅